declfilter/declfilter.go

```diff
@@ -3,20 +3,92 @@
 package declfilter
 
 import (
+	"regexp"
+	"strings"
+
 	"github.com/pelletier/go-toml/v2"
 )
 
-// FilterConfig defines the format of the toml files
-//
-// TODO: currently the only way to "wildcard" a config is to not have a toml
-// file. Should have something more flexible like glob patterns.
-type FilterConfig struct {
+// TODO: describe semantic interpretation for stringSet.
```

Comment:
This seems pretty easy to fix with a brief comment - it's a set of strings built up by interpreting each operation left to right (the operations themselves are relatively self explanatory from the names).